packages/client/manifest.json

```diff
@@ -6723,7 +6723,22 @@
     "illegalChildren": ["section", "sidepanel"],
     "showEmptyState": false,
     "draggable": false,
-    "info": "Side panels are hidden by default. They will only be revealed when triggered by the 'Open Side Panel' action."
+    "info": "Side panels are hidden by default. They will only be revealed when triggered by the 'Open Side Panel' action.",
+    "sendEvents": true,
+    "settings": [
+      {
+        "type": "boolean",
+        "key": "clickOutsideToClose",
+        "label": "Click outside to close",
+        "defaultValue": true
+      },
+      {
+        "type": "event",
+        "key": "sidePanelClose",
+        "label": "Side Panel Close",
+        "info": "Side panel actions configured here will run after the 'Open side panel' action runs, and are not capable of preventing or stopping it. Any form validation should therefore be done before that action is invoked, and not here."
```

Comment:
NAB: This text is a bit wordy/technical. I'm not sure if users would broadly understand the term *invoke* for example.

> Side panel actions configured here will run after the 'Open side panel' action runs, and are not capable of preventing or stopping it.

Is this such a common scenario that it needs to be explained. Would a paragraph in the docs be a more appropriate place for this?